src/coreclr/jit/lower.cpp

```diff
@@ -520,6 +520,10 @@ GenTree* Lowering::LowerNode(GenTree* node)
             LowerJmpMethod(node);
             break;
 
+        case GT_SWIFT_ERROR_RET:
+            LowerNode(node->gtGetOp1());
+
+            FALLTHROUGH;
         case GT_RETURN:
```

Comment:
```suggestion
        case GT_SWIFT_ERROR_RET:
        case GT_RETURN:
```

Otherwise we invoke lowering of op1 twice.

Reply:
`LowerRet` should probably be changed to take `GenTreeOp*` instead.